data.json

```
@@ -139,6 +139,20 @@
         "issue_tracker": "JPS",
         "aliases": ["jps"]
     },
+    {
+        "name": "IdeaVim",
+        "icon_path": "icon_IdeaVim.png",
+        "emoji_name": "ideavim",
+        "role_name": "IdeaVim",
+        "channel_name": "ideavim",
+        "category_name": "Open Source",
+        "subreddit": "",
+        "github": "https://github.com/JetBrains/ideavim",
```

```suggestion
        "github": "ideavim",
```
The bot will automatically add the org prefix :)